README.md

```diff
@@ -1,8 +1,35 @@
 # local-storage-operator
 Operator for local storage
 
+```mermaid
+graph LR
+    PVC -->|Requests local storage using a StorageClass| StorageClass
+    StorageClass -->|Defines a LocalVolume| LocalVolume
```

Comment:
StorageClass is just for reference and is not really used anywhere in PV creation / deletion / binding. The don't define anything.

Reply:
Thanks for reviewing/clarifying. This should be fixed in https://github.com/openshift/local-storage-operator/commit/f8acb21ec74b51b9fdfde6ab9f8c0a8d71ad3ea9